install/kubernetes/cilium/charts/hubble-ui/templates/ingress.yaml

```diff
@@ -1,6 +1,10 @@
 {{- if .Values.ingress.enabled -}}
 {{- $ingressPath := .Values.ingress.path -}}
+{{- if and (eq .Capabilities.KubeVersion.Major "1") (ge .Capabilities.KubeVersion.Minor "14") }}
```

Comment:
This condition is great, however, it might fail in current CI test. Related discussion is here https://github.com/cilium/cilium/pull/12409#discussion_r453295619. I am not sure if it's decided, might need someone to chime in.

Reply:
I have pushed a fix for this in #12409